graph-manager-docs/source/integrations.md

```diff
@@ -162,24 +161,42 @@ Once you've turned on the integration in Datadog, visit the "Integrations" tab i
+In the [Datadog metrics explorer](https://app.datadoghq.com/metric/explorer):
+1. Select `apollo.engine.operations.latency.95percentile` under "Graph"
+2. Select the targeted service name under "Over"
+3. Select `variant` under “One graph per” and choose the two variants for production and staging
+4. Select `Average of reported values` under "On each graph, aggregate with the"
+
+At Apollo, we use Graph Manager to monitor Graph Manager itself, so this graph for us looks like the following:
 
 ![Compare p95](./img/datadog/datadog.png)
 
-_To perform more advanced manipulation of metrics, open up the [Metrics notebook](https://app.datadoghq.com/notebook)._
+To perform more advanced manipulation of metrics, open up a [Datadog notebook](https://app.datadoghq.com/notebook).
+
+#### Alerting with Datadog
 
-#### Monitoring with Datadog
+Graph Manager supports direct alerting on metrics via the Notifications feature, but Datadog can enable more complex alerts via [Datadog monitors](https://docs.datadoghq.com/monitors/).
 
-All of the metrics reported to Datadog can be notified on directly through Graph Manager via the Notifications feature, but Datadog can be a powerful partner in enabling more complex alerts.
+**Example**: Graph Manager's Notifications feature supports alerts that trigger when the percentage of requests with an error in the last 5 minutes exceeds some threshold for a specific operation. Suppose that instead of alerting on a specific operation in the last 5 minutes, we want to alert on the error percentage over all operations in some graph in the last 10 minutes, such as when the percentage exceeds 1% for a graph `mygraph` with variant `staging`.
 
-**Example**: Suppose you have a query that is run against your GraphQL server with a much higher volume in the morning than in the afternoon. You want to enable monitoring on that query's latency and error rates, but if the query volume is very low, you have a higher tolerance for latency and one error will skew the error rate and make the monitor too noisy.
+The [Datadog metric alert query](https://docs.datadoghq.com/api/?lang=curl#metric-alert-query) needed here is
+```
+sum(last_10m):sum:apollo.engine.operations.error_count{service:mygraph,variant:staging}.as_count().rollup(sum).fill(null) / sum:apollo.engine.operations.count{service:mygraph,variant:staging}.as_count().rollup(sum).fill(null) > 0.01
+```
+The `.rollup(sum).fill(null)` is necessary since `apollo.engine.operations.count` is a [Datadog gauge](https://docs.datadoghq.com/developers/metrics/types/), which means it [defaults to using `avg` for time aggregation]() and [defaults to linear interpolation during space aggregation and query arithmetic](https://docs.datadoghq.com/monitors/guide/monitor-arithmetic-and-sparse-metrics/). The `.as_count()` is necessary to ensure that [operation counts are summed before the division and not after](https://docs.datadoghq.com/monitors/guide/as-count-in-monitor-evaluations/).
```

Comment:
It might be a bit over-kill to include the detailed breakdown of using `.as_count()` and `.rollup(sum)`, but I suppose correctness is better. It just seems to point out some product usability issues in Datadog.

Reply:
My worry is they won't understand why I'm adding those functions, and just not include them in their own monitors. The use of `as_count()` is particularly non-intuitive, since the [Datadog docs](https://docs.datadoghq.com/graphing/functions/rollup/) misleadingly state "these functions have no effect for other metric types."

The docs are partially right here, in that `as_count()` does not change the default time aggregation function and `fill()` interpolation options for gauges. The docs are also partially wrong here, since `as_count()` does still trigger swapping of monitor aggregation and query arithmetic.